docs/core/whats-new/whats-new-in-dotnet-core-2-0.md

```diff
@@ -0,0 +1,138 @@
+
+.NET Standard 2.0 class libraries also can reference .NET Framework class libraries, provided that they call APIs that are present in the .NET Standard 2.0. No recompilation of the .NET Framework libraries is required.
+
+**Expanded surface area**
+
+The total number of APIs available on .NET Core 2.0 has more than doubled in comparison with .NET Core 1.1. 
+
+**Support for .NET Framework libraries**
+
+.NET Core code can reference existing .NET Framework libraries, including existing NuGet packages. Note that the libraries must use APIs that are found in .NET Standard.
+
+## Visual Studio integration
+
+Visual Studio 2017 Preview 15.3 and in some cases Visual Studio for Mac offer a number of significant enhancements for .NET Core developers.
+
+**Retargeting to .NET Core 2.0**
+
+If the .NET Core 2.0 SDK is installed, projects that target .NET Core 1.x can be retargeted to .NET Core 2.0. To do this, you open the **Application** tab of the project's properties dialog and change the **Target framework** value to **.NET Core 2.0**. You can also change it by editing your project file and changing the value of the `<TargetFrame>` element from 1.x to 2.0:
+
+```xml
+<PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+ </PropertyGroup>
+```
+
+**Live Unit Testing support for .NET Core**
+
+Whenever you modify your code, Live Unit Testing automatically runs any affected unit tests in the background and displays the results and code coverage live in the Visual Studio environment. .NET Core 2.0 now supports Live Unit Testing. Previously, Live Unit Testing was available only for .NET Framework applications. 
+
+For more information, see [Live Unit Testing with Visual Studio 2017](https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing) and the [Live Unit Testing FAQ](https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing-faq).
+
+**Better support for multiple target frameworks**
+
+If you are building a project for multiple target frameworks, you can now select the target platform from the top-level menu. In the following figure, a project named SCD1 targets 64-bit Mac OS X 10.11 (`osx.10.11-x64`) and 64-bit Windows 10/Windows Server 2016 (`win10-x64`). You can select the target framework before selecting the project button, in this case to run a debug build.
+
+![Selecting the target framework when building a project](multitarget.png) 
+
+**Side-by-side support for .NET Core SDKs**
+
+You can now install the .NET Core SDK independently of Visual Studio. This makes it possible for a single version of Visual Studio to build projects that target different versions of .NET Core. Before Visual Studio 2017 Preview 15.3, Visual Studio and the .NET Core SDK were tightly coupled; a particular version of the SDK accompanied a particular version of Visual Studio.
```

Comment:
well, if you install the workload, you get the 1.1. version of the SDK installed. should we note that? /cc @morrisjoe